CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # CHANGELOG
 
 ## Version 1.0.10
+- Fixed Exceeded property length leads to lost event.
+- Fixed issue of sending custom event.
+- Method `sanitizeStringForJSON` takes string input and converts it to JSON friendly string.
+- Class `com.microsoft.applicationinsights.telemetry.SanitizationUtils` has methods to sanitize string for JSON.
+- Fixed Request Telemetry Sending bug with new schema.
```

Comment:
when Sending starts with upper case letter? More details would be benefitial (maybe at least issue number)